backend/crud.py

```diff
@@ -437,18 +438,24 @@ def _query_entity_with_filters(filters: dict, schema: Schema, all: bool = False,
     '''
     attrs = {i.attribute.name: i.attribute
              for i in schema.attr_defs if i.attribute.type.value.filters}
-    attrs_filters, name_filters = _parse_filters(filters=filters, attrs=attrs.keys())
+    attrs_filters, entity_filters = _parse_filters(filters=filters, attrs=attrs.keys())
```

Comment:
Above, this was renamed to `attr_filters` (singular first part).

While it's certainly not necessary, would it make sense to do the same here?

Reply:
Yes, indeed. I had also considered changing it, but resisted the urge to refactor. But now that you mention it, let me get on it.